connect/api/connect.api

```diff
@@ -14,14 +14,6 @@ public final class com/stripe/android/connect/AccountOnboardingProps$Creator : a
 	public synthetic fun newArray (I)[Ljava/lang/Object;
 }
 
-public final class com/stripe/android/connect/AccountOnboardingView : com/stripe/android/connect/StripeComponentView, com/stripe/android/connect/webview/StripeConnectWebViewContainer {
```

Comment:
👀 These component view APIs should have been removed in https://github.com/stripe/stripe-android/pull/10341. Cleaned them up in this PR while I was in the area.